tests/int/test-just_install_functions.bsh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+# Note all just_install_functions.bsh test are currently enabled only for windows.  Tests currently fail on most linux test images, as the installed tools often rely on glibc and a valid download utility (e.g., wget).
+
+if [ -z ${VSI_COMMON_DIR+set} ]; then
+  VSI_COMMON_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.."; pwd)"
+fi
+
+source "${VSI_COMMON_DIR}/tests/testlib.bsh"
+source "${VSI_COMMON_DIR}/linux/just_files/just_install_functions.bsh"
+
+# test cmake install
+[ "${OS-}" = "Windows_NT" ] || skip_next_test
```

Comment:
Why only in windows?

Reply:
Check for curl/wget instead, like `command -v curl &> /dev/null || command -v wget &> /dev/null || skip_next_test`